src/FINDOLOGIC/Api/Responses/Xml21/Properties/Filter.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace FINDOLOGIC\Api\Responses\Xml21\Properties;
+
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
+use SimpleXMLElement;
+
+class Filter
+{
+    /** @var int|null $itemCount */
+    private $itemCount;
+
+    /** @var string|null $cssClass */
+    private $cssClass;
+
+    /** @var string|null $noAvailableFiltersText */
+    private $noAvailableFiltersText;
+
+    /** @var string $name */
+    private $name;
+
+    /** @var string $display */
+    private $display;
+
+    /** @var string $select */
+    private $select;
+
+    /** @var int|null $selectedItems */
+    private $selectedItems;
+
+    /** @var string $type */
+    private $type;
+
+    /** @var Attributes|null $attributes */
+    private $attributes;
+
+    /** @var Item[] $items */
+    private $items = [];
+
+    public function __construct(SimpleXMLElement $response)
+    {
+        $this->itemCount = ResponseHelper::getIntProperty($response, 'itemCount');
```

Comment:
If `itemCount` was `0` it would be set to `null`. Submit as third param `true` to ensure the value "0" is allowed there.

Reply:
Done